lib/Cleantalk/Custom/Antispam/CleantalkResponse.php

```diff
@@ -0,0 +1,181 @@
+<?php
```

Comment:
Why this class is in `Custom` namespace? Is not it really enough the common class?

Reply:
I'll try to solve